internal/config/config.go

```diff
@@ -236,8 +236,8 @@ func Initialize(configFiles ...string) {
 					config.Kafka.CACert = *b.Cacert
 				}
 				if b.Sasl != nil {
-					if b.Sasl.SecurityProtocol != nil && *b.Sasl.SecurityProtocol != "" {
-						config.Kafka.SASL.SecurityProtocol = *b.Sasl.SecurityProtocol
+					if b.SecurityProtocol != nil && *b.SecurityProtocol != "" {
+						config.Kafka.SASL.SecurityProtocol = *b.SecurityProtocol
```

Comment:
This was deprecated in clowder and moved up one level, linter errors on deprecations.